topics/fair/tutorials/ro-crate-in-python/tutorial.md

```diff
@@ -256,8 +271,11 @@ You can fetch an entity by its `@id` as follows:
 article = crate.dereference("paper.pdf")  # or crate.get("paper.pdf")
 ```
 
+# Command Line Interface
 
-## Command Line Interface
+> <comment-title>Jupyter Notebook users: switch to a terminal</comment-title>
+> The code cells in this section use Unix shell commands, which can't be run within a notebook. Open a Unix/Linux terminal to follow along.
+{: .comment}
 
 `ro-crate-py` includes a hierarchical command line interface: the `rocrate` tool. `rocrate` is the top-level command, while specific functionalities are provided via sub-commands. Currently, the tool allows to initialize a directory tree as an RO-Crate (`rocrate init`) and to modify the metadata of an existing RO-Crate (`rocrate add`).
```

Comment:
We could consider splitting the tutorial in two here, and then use a `bash` notebook for the second half with the cli. @simleo @stain what do y'all think?

Reply:
I'm not sure. One alternative would be to add leading "!" to the commands, but then it will read too jupyter-specific. Would the split involve spreading the tutorial over two separate pages? That would make it a bit harder to follow.